polyfills.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * These functions are either introduced in WP Core in latest version or
+ * available in different versions. For simplicity for future, we can remove them
+ * if they are supported in WP Core versions which this plugin supports.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+// WP Filesize function.
+if ( ! function_exists( 'wp_filesize' ) ) {
+	/**
+	 * Wrapper for PHP filesize with filters and casting the result as an integer.
+	 *
+	 * This function is introduced in WP 6.0, to support backward compatibility
```

Comment:
```suggestion
	 * This function was introduced in WP 6.0, for backward compatibility the
```

Reply:
Let's also split the sentence, since it's more like 2 separate sentences.
```suggestion
	 * This function was introduced in WP 6.0. To support backward compatibility, the
```